tools/distrib/python/make_grpcio_tools.py

```diff
@@ -200,6 +200,8 @@ def _bazel_name_to_file_path(name):
             if (
                 "/upb_generator/minitable/main.cc" in filepath
                 or "/upb_generator/c/generator.cc" in filepath
+                or "/protobuf/src/google/protobuf/compiler/main_no_generators.cc"
+                in filepath
```

Comment:
```suggestion
                or "/protobuf/compiler/main_no_generators.cc" in filepath
```
actually since we're checking `in filepath`, I'd rather not include `protobuf/src/google`